### DISCUSSION TITLE
Trying to use Spatie Translatable plugin fires errors

### DISCUSSION THREAD

**Discussion by faridsa** [filamentphp/filament#6803] [Help]
After adding `use Filament\Resources\Concerns\Translatable;` to my models and `use CreateRecord\Concerns\Translatable;` to my resources, then I got the following fatal error:

`Cannot make static method Filament\Resources\Resource::getRecordTitle() non static in class Filament\Resources\Pages\CreateRecord\Concerns\Translatable`

**Comment by danharrin** [comment]
Sounds like you have the wrong traits on the wrong classes, the resource trait should be on the resource class, the page traits should be on the corresponding page classes
